content/javascript/concepts/storage/terms/setItem.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: '.setItem()' 
+Description: 'updates the value to the specified storage object key if exists, else adds the key to the specified storage object.' 
+Subjects: 
+  - 'Web Development'
+  - 'Computer Science'
+Tags: 
+  - 'Storage'
+  - 'Objects'
+  - 'Methods'
+CatalogContent: 
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`.setItem()`** method updates the `value` to the specified storage object `key` if exists, else adds the `key` to the specified storage object and returns `undefined`.
```

Comment:
```suggestion
The **`.setItem()`** method updates the value to the specified storage object key, if it exists, else adds the key to the specified storage object and returns `undefined`.
```

Reply:
backticks are used for specific terms or elements within a code block or the language generally. However, generic terms such as class or string don't need to be formatted.

Reply:
okay